README.md

```diff
@@ -1,3 +1,11 @@
+# Deprecated
+
+As of September 2024, this repo and the [npm package](https://www.npmjs.com/package/@nypl/dgx-header-component), are deprecated. See the instructions in the [`nypl-header-app`](https://github.com/NYPL/nypl-header-app) repo for using the new NYPL Header and Footer.
+
+Please contact the NYPL Design System team or Remediation Team for more information.
```

Comment:
Do you think anyone reading this would know how to reach these teams?

Reply:
I think it'll be OK just to leave this line out.  The team names are probably going to change and the link to the active replacement is enough.  (If that gets deprecated, it'll have a link to whatever replaces it, etc)